### PR TITLE
fix #80446 www.news18.com

### LINKED ISSUE #80446: www.news18.com
### Issue URL (Ads)
 [https://www.news18.com/news/india/if-pakistan-attacks-with-one-nuclear-bomb-india-can-finish-us-with-20-pervez-musharraf-2047265.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.news18.com%2Fnews%2Findia%2Fif-pakistan-attacks-with-one-nuclear-bomb-india-can-finish-us-with-20-pervez-musharraf-2047265.html)
### Comment
Username: @chirag127
> leftover + share widget

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>
<details>
  <summary>Screenshot 4</summary>

  Screenshot 4
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances

### PR DESCRIPTION
**Pull request by stanislav-atr**
#80446

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -305,6 +305,7 @@ cointelegraph.com##.banner-blocked
 ||litecoin-faucet.com/assets/img/static/
 litecoin-faucet.com##.spot
 litecoin-faucet.com##.sticky-top
+news18.com##.article_youtube_player
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/specific.txt:308]: I don't see this element

File: MobileFilter/sections/specific_web.txt [hunk 2]
```diff
@@ -114,6 +114,7 @@ m.sobaka.ru##.b-post-view__banner
 m.sobaka.ru##.b-banner-point
 sportskeeda.com##.sticky-container
 m.cyber.sports.ru##.zen-adload-ad-wrapper
+news18.com##.article_main > div[style^="margin-bottom: 15px; background: rgb"]
```
Comment (Sergey-Lyapin) [inline MobileFilter/sections/specific_web.txt:117]: please add rules for other leftovers too

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

**Review by Alex-302 — CHANGES_REQUESTED** [review]
missed social leftover `news18.com##.article_socials`

**Comment by Alex-302** [comment]
image
also ad leftover

**Comment by stanislav-atr** [comment]
I've added `news18.com##div[class^="jsx-"][style^="min-height: 1170px"]` with `style^=` as it ends with semicolon on my end, on and Alex's screenshot there is none.
<details><summary>Screenshot:</summary>

image
</details><br/>

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**